### PR TITLE
Handle access to vssps for PerfStar

### PR DESCRIPTION
**Pull request by YuliiaKovalova**
## Fixes:
Issues with access to msbuild.exe bit for perfstar run.
In order to secure that, a new SP was created and added to the corresponding organization.

## Validation:
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9874247&view=logs&j=063f674e-38db-59b9-8dc6-bda499493784&t=d2d8d882-24a4-5280-d5d8-e137cda2c3ba

### CODE REVIEW

**Review by JanKrivanek — APPROVED** [review]
:clap:

**Review by rainersigwald — APPROVED** [review]
I assume that failure is one where we need to refactor to deal with #10299?

**Comment by YuliiaKovalova** [comment]
> I assume that failure is one where we need to refactor to deal with #10299?

I am not sure it's related.
Hopefully, @AR-May has more insights :)

File: .exp-insertions.yml [hunk 1]
```diff
@@ -155,6 +155,21 @@ jobs:
...
+    displayName: 'Set AzDO.DotnetPerfStarToken'
+    enabled: true
+    inputs:
+      azureSubscription: 'dotnet-perfstar at app.vssps.visualstudio.com'   # Azure DevOps service connection
+      scriptType: 'pscore'
+      scriptLocation: 'inlineScript'
+      inlineScript: |
+        # '499b84ac-1321-427f-aa17-267ca6975798' for Azure DevOps
+        $token = az account get-access-token --query accessToken --resource 499b84ac-1321-427f-aa17-267ca6975798 -o tsv
+        Write-Host "Setting AzDO.DotnetPerfStarToken: ${token}"
```
Comment (AR-May) [inline .exp-insertions.yml:170]:
```suggestion
        Write-Host "Setting AzDO.DotnetPerfStarToken"
```

**Review by AR-May — CHANGES_REQUESTED** [review]

**Comment by AR-May** [comment]
> > I assume that failure is one where we need to refactor to deal with #10299?
> 
> I am not sure it's related. Hopefully, @AR-May has more insights :)

No, this is unrelated. This is for being able to run msbuild.exe in perf tests.

**Review by AR-May — APPROVED** [review]

**Pull request merged**